nextflow.config

```diff
@@ -144,6 +309,12 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Set default registry for Docker and Podman independent of -profile
+// Will not be used unless Docker / Podman are enabled
+// Set to your registry if you have a mirror of containers
+docker.registry = 'quay.io'
+podman.registry = 'quay.io'
```

Comment:
Can you add singularity.registry = 'quay.io' too?
You might need to bump the minimal nextflow version

Reply:
Fixed in #68